FMS/fms/fms.F90

```diff
@@ -299,7 +299,7 @@ module fms_mod
 
 !   ---- private data for check_nml_error ----
 
-   integer, private :: num_nml_error_codes, nml_error_codes(20)
+   !integer, private :: num_nml_error_codes, nml_error_codes(20)
```

Comment:
ah a variable that is only written but not set or used otherwise? Good find!

Reply:
Yeah, this was crashing the code when running with debug options, since the array index had bogus values (uninitialized).